src/app/pages/flex-page/blocks/HeroBlock.scss

```diff
@@ -0,0 +1,39 @@
+@import 'pattern-library/core/pattern-library/headers';
+
+.flex-page.page > section.content-block-hero {
+    width: 100%;
+    margin: 0;
+
+    > div.hero-inner-wrapper {
+        /* to show above divider images */
+        position: relative;
+        z-index: 1;
+
+        display: grid;
+        grid-template: 1fr / 1fr 1fr;
```

Comment:
For mobile: `grid-template-columns: 1fr` and `text-align: center`
hero-content should have `max-width: $text-content-max` and `margin: auto`

Reply:
@RoyEJohnson this comment isn't necessary anymore right? you added the mobile switch?